app2.js

```diff
@@ -0,0 +1,79 @@
+// Get the form and tbody elements 
+const form = document.querySelector('#form')
+const tableBody = document.querySelector('#table-body')
+
+// Add a new book in the table
+function addBookToUI(book){
+  const tr = document.createElement('tr')
+  tr.innerHTML=
+  `
+  <td>${book.title}</td>
+  <td>${book.author}</td>
+  <td>${book.isbn}</td>
+  <td id="delete"><i class="fa-solid fa-trash text-danger delete" style="cursor:pointer"></i></td>
+  `
+  tableBody.appendChild(tr)
+
+  // Delete a book from the table
+  const delBtn = tr.querySelector('#delete') 
```

Comment:
Take this functionality out of the addBookToUI function. There is no need to have a delete function inside an add book function